docs/changelog/86647.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 86647
```

Comment:
Should this be mentioned here? I don't think this changes anything with the tsdb feature flag in place? If so then we should label this PR also as non-issue.

Reply:
Silly generated thing....